drivers/volume/portworx/portworx.go

```diff
@@ -6316,4 +6317,70 @@ func (d *portworx) UpdateSkinnySnapReplNum(repl string) error {
 		break
 	}
 	return nil
-}
\ No newline at end of file
+}
+
+// CreateDefragSchedule create defrag schedule for provided defrag job
+func (d *portworx) CreateDefragSchedule(startTime string, defragJob *api.DefragJob) error {
```

Comment:
We can return the `DefragSchedule` object here, that way the function can be used in tests where we want to `edit` or `delete` the schedule if needed.

Reply:
Yeah right, Incorporated it.